modules/tinymce/src/themes/silver/test/ts/browser/skin/OxideColorSwatchMenuTest.ts

```diff
@@ -149,6 +175,57 @@ describe('browser.tinymce.themes.silver.skin.OxideColorSwatchMenuTest', () => {
     closeSwatchButtonMenu();
   });
 
+  it('TINY-9395: Check structure of menu color swatch', async () => {
```

Comment:
It's probably a good idea to have a test that checks that the focus starts on something that isn't just going to generally be a default colour.

Reply:
So maybe also include a test with keyboard navigation.

Reply:
Since it's reliant on the color-detection, and the color-detection is already tested in other tests, I didn't think it'd be needed to be given a through test. It detects the default black color, which should confirm that it does detect colors, the rest being tested elsewhere. Similar with the keyboard navigation.

But if you think it'd still be best to include these in tests, I can do that.

Reply:
Having test gaps because of "known implementation details" causes a problem when people try to refactor things later, so just bear that in mind. It would be extremely easy to break this for example. And adding a test is not a lot of effort for the future pain it might avoid (IMO)

Reply:
Might been a misunderstanding, but I added some more tests as requested.

Reply:
Seems reasonable. Next time, you can check that `<right>` works as expected as well :)